packages/@react-types/shared/src/inputs.d.ts

```diff
@@ -62,6 +62,7 @@ export interface RangeInputBase<T> {
 }
 
 export interface HelpTextProps {
+  validationState?: ValidationState,
```

Comment:
Missing a description here, perhaps:
```suggestion
  /** Whether the field should display the description or error message. */
  validationState?: ValidationState,
```
Might get overwritten by Validation's validationState description for the SpectrumHelpTextProps but that might be ok